clinvar_parser/clinvar-parser/src/main/java/uk/ac/ebi/eva/clinvar/PublicSetParser.java

```diff
@@ -0,0 +1,27 @@
+package uk.ac.ebi.eva.clinvar;
+
+import uk.ac.ebi.eva.clinvar.model.ClinvarSet;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBElement;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+import javax.xml.transform.stream.StreamSource;
+import java.io.StringReader;
+
+class PublicSetParser {
+
+    Unmarshaller unmarshaller;
```

Comment:
Is this supposed to be accessible by other classes? Otherwise make it `private`.